test/snippet/io/sequence_file/sequence_file_output_fields_trait_1.cpp

```diff
@@ -0,0 +1,30 @@
+#include <sstream>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/alphabet/quality/phred42.hpp>
+#include <seqan3/alphabet/quality/qualified.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+
+int main()
+{
+    using seqan3::operator""_dna5;
+    using seqan3::operator""_phred42;
+
+    seqan3::sequence_file_output fout{std::ostringstream{}, seqan3::format_fasta{}, seqan3::fields<seqan3::field::ID, seqan3::field::SEQ_QUAL>{}};
+
+    for (int i = 0; i < 5; i++)
+    {
+        std::string id{"test_id"};
+        // vector of combined data structure:
+        std::vector<seqan3::qualified<seqan3::dna5, seqan3::phred42>> seq_qual{seqan3::qualified<seqan3::dna5, seqan3::phred42>{'N'_dna5, '7'_phred42}};
```

Comment:
line length

Reply:
I really think in some of these cases, making it < 120 lines makes it much less readable... Ex:

```cpp
        std::vector<seqan3::qualified<seqan3::dna5,
                                      seqan3::phred42>> seq_qual{seqan3::qualified<seqan3::dna5,
                                                                                   seqan3::phred42>{'N'_dna5,
                                                                                                    '7'_phred42}};
```